messaging-experiments.yaml

```diff
@@ -200,7 +200,7 @@
               label:
                 string_id: cfr-doorhanger-extension-sumo-link
               sumo_path: extensionrecommendations
-            text: You’re using the only major browser backed by a non-profit. See how Firefox puts your privacy first.
+            text: 'You’re using the only major browser backed by a non-profit. See how Firefox puts your privacy first.'
```

Comment:
How about using double quotes?

Reply:
Quotes don't seem to help, I can fix this when publishing to RS. I dropped the quotes entirely.